src/renderer/src/views/Home/index.tsx

```diff
@@ -34,37 +39,84 @@ const BottomBarGridSection = styled.div(`
 
 export type PanelName = 'territory' | 'observations' | 'sync' | 'settings'
 
-interface Props {
+type LocationState = {
+  defaultTab?: PanelName
   showBottomBar?: boolean
 }
 
-type LocationState = { defaultTab: PanelName }
+export const Home = () => {
+  const location = useLocation()
+  const state = location.state as LocationState
+  const [activePanel, setPanelName] = React.useState<PanelName>(state?.defaultTab || 'territory')
+  const intl = useIntl()
+  const [snackbarMessageOpen, setSnackbarMessageOpen] = React.useState(!!state.showBottomBar)
+  const projectName = useMapeoDeviceStore((store) => store.projectName)
 
-export const Home = ({ showBottomBar }: Props) => {
-  const { state } = useLocation()
-  const tabState = state as LocationState
-  const [activePanel, setPanelName] = React.useState<PanelName>(tabState?.defaultTab || 'territory')
+  console.log({ snackbarMessageOpen })
 
   return (
-    <Box minHeight="100vh" display="flex" flex={1}>
-      <GridContainer>
-        <SidebarGridSection>
-          <Sidebar activeTab={activePanel} onChangeTab={setPanelName} />
-        </SidebarGridSection>
-        <MainGridSection>
-          <TabPanel active={activePanel === 'territory'}>Territory</TabPanel>
-          <TabPanel active={activePanel === 'observations'}>Observations</TabPanel>
-          <TabPanel active={activePanel === 'sync'}>Sync</TabPanel>
-          <TabPanel active={activePanel === 'settings'}>
-            <Settings />
-          </TabPanel>
-        </MainGridSection>
-        {showBottomBar && (
-          <BottomBarGridSection>
-            <div style={{ height: 30 }} />
-          </BottomBarGridSection>
-        )}
-      </GridContainer>
-    </Box>
+    <>
+      <Box minHeight="100vh" display="flex" flex={1}>
+        <GridContainer>
+          <SidebarGridSection>
+            <Sidebar activeTab={activePanel} onChangeTab={setPanelName} />
+          </SidebarGridSection>
+          <MainGridSection>
+            <TabPanel active={activePanel === 'territory'}>
+              {intl.formatMessage(sidebarMessages.territory)}
+            </TabPanel>
+            <TabPanel active={activePanel === 'observations'}>
+              {intl.formatMessage(sidebarMessages.observations)}
+            </TabPanel>
+            <TabPanel active={activePanel === 'sync'}>{intl.formatMessage(sidebarMessages.sync)}</TabPanel>
```

Comment:
was wondering if it would make sense to leave these panels blank. thoughts?

Reply:
We should probably just add static screen shots so it looks like the app

Reply:
**screen shots of the actual tabs in Mapeo

Reply:
okay to leave as a nice-to-have follow-up item then?

Reply:
yeah...ill do it today, but lets not block this PR